nodes.py

```diff
@@ -309,3 +409,28 @@ def sample(
     "KRestartSamplerAdv": "KSampler With Restarts (Advanced)",
     "KRestartSamplerCustom": "KSampler With Restarts (Custom)",
 }
+
+if os.environ.get("COMFYUI_RESTART_SAMPLING_SELFTEST", "").strip() == "1":
```

Comment:
Did you mean to replace this with the variable you defined at the top?

Reply:
good point! i forgot to do that somehow.